ql/src/test/org/apache/hadoop/hive/metastore/txn/TestCompactionTxnHandler.java

```diff
@@ -86,14 +86,14 @@ public void testFindNextToCompact() throws Exception {
     rqst.setPartitionname("ds=today");
     txnHandler.compact(rqst);
     long now = System.currentTimeMillis();
-    CompactionInfo ci = txnHandler.findNextToCompact("fred");
+    CompactionInfo ci = txnHandler.findNextToCompact("fred", "4.0.0");
```

Comment:
could you please extract version to constants

Reply:
done